ang/crmMosaico/BlockSchedule.html

```diff
@@ -4,12 +4,15 @@
 
   <div class="crmMosaico-schedule-inner form-inline form-inline-tabs" ng-init="selectedTab = 'sendNow'">
     <div class="form-group" ng-class="{'active':schedule.mode === 'now'}" ng-click="selectedTab = 'sendNow'; schedule.mode = 'now'">
-      <input ng-model="schedule.mode" type="radio" name="send" value="now" id="schedule-send-now" ng-checked="selectedTab === 'sendNow'" />
-      <label for="schedule-send-now">{{ts('Send immediately')}}</label>
-    </div>
-    <div class="form-group" ng-class="{'active':schedule.mode === 'at'}" ng-click="selectedTab = 'sendAt'; schedule.mode = 'at'">
-      <input ng-model="schedule.mode" type="radio" name="send" value="at" id="schedule-send-at" ng-checked="selectedTab === 'sendAt'" />
-      <label for="schedule-send-at">{{ts('Send on..')}}</label>
+      <div class="radio">
+        <input ng-model="schedule.mode" type="radio" name="send" value="now" id="schedule-send-now" ng-checked="selectedTab === 'sendNow'" />
+        <label for="schedule-send-now">{{ts('Send immediately')}}</label>
+      </div>
```

Comment:
Hey @boriszaydlin. 
This is intentional to resolve the small space issue between form groups. We agreed on this solution after a discussion with Alessandro and based on the solution mentioned here 

https://css-tricks.com/fighting-the-space-between-inline-block-elements/